### PR TITLE
Fix powershell cat invocation to ensure correct encoding and line ending.

### PR DESCRIPTION
**Pull request by aam**
Without these changes I couldn't get `dev\bots\test.dart` running on my Windows box.

Not sure what configuration was applied to AppVeyor to make it work without these changes. It might be that there are some cygwin tools(cat) installed.

### CODE REVIEW

**Comment by aam** [comment]
@goderbauer I think you were not able to run `dev\bots\test.dart` on your Windows box in the past neither. Can you give this a try please?

File: dev/bots/test.dart [hunk 1]
```diff
@@ -76,8 +76,8 @@ Future<Null> _verifyInternationalizations() async {
   final String localizationsFile = path.join('packages', 'flutter_localizations', 'lib', 'src', 'l10n', 'localizations.dart');
 
   final EvalResult sourceContents = await _evalCommand(
-    'cat',
-    <String>[localizationsFile],
+    'powershell',
```
Comment (goderbauer) [inline dev/bots/test.dart]: This will not work on mac/linux now?

Comment (aam) [reply dev/bots/test.dart]: Oops, yes, this has to be guarded as OS(windows)-specific

File: dev/bots/test.dart [hunk 2]
```diff
@@ -75,11 +75,18 @@ Future<Null> _verifyInternationalizations() async {
 
   final String localizationsFile = path.join('packages', 'flutter_localizations', 'lib', 'src', 'l10n', 'localizations.dart');
 
-  final EvalResult sourceContents = await _evalCommand(
-    'powershell',
-    <String>['\$PSDefaultParameterValues["*:Encoding"]="utf8";(gc $localizationsFile) -join "`n"'],
-    workingDirectory: flutterRoot,
-  );
+  final EvalResult sourceContents =
```
Comment (goderbauer) [inline dev/bots/test.dart]: nit: Since each branch is multiple lines I'd almost prefer a real if/else to make it easier to read...

**Review by goderbauer — APPROVED** [review]
LGTM, thanks for looking into it and fixing it.

**Review by goderbauer — APPROVED** [review]
LGTM

**Pull request merged**